com.unity.render-pipelines.universal/Runtime/UniversalRenderPipelineCore.cs

```diff
@@ -440,6 +427,15 @@ void SortCameras(Camera[] cameras)
                 desc.msaaSamples = msaaSamples;
                 desc.sRGB = (QualitySettings.activeColorSpace == ColorSpace.Linear);
             }
+            else
+            {
+                desc = camera.targetTexture.descriptor;
+                // SystemInfo.SupportsRenderTextureFormat(camera.targetTexture.descriptor.colorFormat)
```

Comment:
The commentary is not necessary for the code. We should pursue with foundation team the behaviour of `SystemInfo.SupportsRenderTextureFormat(camera.targetTexture.descriptor.colorFormat)`